(WIP) to_padded_tensor() triton kernel for NJT

jbschlosser opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #121947
* #126552
* #125947

DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

jbschlosser committed:
(WIP) to_padded_tensor() triton kernel for NJT

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

[ghstack-poisoned]

clee2000 commented:
@clee2000 has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D55662628).

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser commented:
@jbschlosser has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D55662628).

jbschlosser committed:
Update

[ghstack-poisoned]

jbschlosser committed:
Update

[ghstack-poisoned]

jbschlosser committed:
Update

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]

jbschlosser committed:
Update on "(WIP) to_padded_tensor() triton kernel for NJT"


DO NOT REVIEW

Differential Revision: [D55662628](https://our.internmc.facebook.com/intern/diff/D55662628)

[ghstack-poisoned]
